web/src/components/vis/ForceDirectedGraph.vue

```diff
@@ -76,9 +79,20 @@ export default {
         .clamp(true);
     },
   },
+  watch: {
+    search: function highlightSearchedNode(searchedNodes) {
```

Comment:
Prefer ES6 property value shorthand in vue functions.

I appreciate that you documented the function's purpose with its name.  Perhaps just move that into a comment inside the method?

```
watch: {
   search() {
     ...
  },
  ...
}
```